src/main/java/com/luxiergerie/Controllers/ClientController.java

```diff
@@ -0,0 +1,78 @@
+package com.luxiergerie.Controllers;
+
+import com.luxiergerie.DTO.ClientDTO;
+import com.luxiergerie.Domain.Entity.Client;
+import com.luxiergerie.Domain.Entity.Role;
+import com.luxiergerie.Domain.Entity.Room;
+import com.luxiergerie.Domain.Mapper.ClientMapper;
+import com.luxiergerie.Domain.Repository.ClientRepository;
+import com.luxiergerie.Domain.Repository.RoleRepository;
+import com.luxiergerie.Domain.Repository.RoomRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/api/client")
+public class ClientController {
+
+    private final ClientRepository clientRepository;
+    private final RoomRepository roomRepository;
+    private final RoleRepository roleRepository;
+
+    public ClientController(ClientRepository clientRepository, RoomRepository roomRepository, RoleRepository roleRepository) {
+        this.clientRepository = clientRepository;
+        this.roomRepository = roomRepository;
+        this.roleRepository = roleRepository;
+    }
+
+    @GetMapping
+    public List<ClientDTO> getClients() {
+        return this.clientRepository.findAll().stream().map(ClientMapper::toDTO).collect(Collectors.toList());
+    }
```

Comment:
this.clientRepository.findAll().stream()
                                              .map(ClientMapper::toDTO)
                                             .collect(Collectors.toList()) ici enlever le static